openproblems/tasks/_cell_cell_communication/_common/utils.py

```diff
@@ -105,3 +105,19 @@ def map_gene_symbols(adata, map_filename: Union[str, pathlib.Path]):
         uns=adata.uns,
         obsm=adata.obsm,
     )
+
+
+# Join predictions to target
+def join_truth_and_pred(adata):
+    merge_keys = list(adata.uns["merge_keys"])
+    gt = adata.uns["ccc_target"].merge(adata.uns["ccc_pred"], on=merge_keys, how="left")
+
+    gt.loc[gt["response"].isna(), "response"] = 0
+    gt.loc[gt["score"].isna(), "score"] = np.nanmin(gt["score"])
```

Comment:
missing should be less than min(score), no? Maybe let's make it `np.nanmin(gt["score"]) - np.finfo(float).eps`